browser-extension/mv2/src/background/background.js

```diff
@@ -252,6 +252,11 @@ BG.Methods.replaceHeader = function (headers, newHeader) {
   BG.Methods.addHeader(headers, newHeader);
 };
 
+BG.Methods.getHeaderValue = (headers, headerName) => {
+  const header = headers.find(({ name }) => name.toLowerCase() === headerName.toLowerCase());
+  return header?.value;
+};
+
```

Comment:
We should add test case for this function

Reply:
Added test